src/config/security/apparmor.md

```diff
@@ -16,3 +16,6 @@ To use AppArmor on a system, one must:
 
 To accomplish the third step, consult [the documentation on how to modify the
 kernel cmdline](./../kernel.md#cmdline).
+
+AppArmor tools aa-genprof and aa-logprof require either configured
```

Comment:
Both `aa-*` should be in the form `[aa-genprof(8)](https://man.voidlinux/aa-genprof.8)`.